src/server.rs

```diff
@@ -64,7 +64,10 @@ fn main() -> std::io::Result<()> {
                 let mut response = String::new();
                 socket.read_to_string(&mut response)?;
                 run_system_command(&response);
-                log::debug!("Socket: {:?} Address: {:?} Response: {}", socket, address, response);
```

Comment:
I think we should keep this in one line. During race conditions multiple commands can reach us at once and then it'll be hard to figure out which command is in respect to which response.